task8/ServletsOnMaven/src/main/java/ServletsOnPosts/TakeLikeServlet.java

```diff
@@ -0,0 +1,32 @@
+package ServletsOnPosts;
+
+import Posts.UtilClasses.JSONDecorator;
+import WorkWithDB.WorkWithDB;
+import org.apache.commons.io.IOUtils;
+
+import javax.annotation.Resource;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.sql.DataSource;
+import java.io.IOException;
+
+public class TakeLikeServlet extends HttpServlet {
+
+    @Resource(name="jdbc/mydb")
+    private DataSource dataSource;
+
+    static class PairForLike{
```

Comment:
Should this class be package-private? Seems like it would get on with the private visibility